Makefile

```
@@ -390,7 +391,8 @@ $(FCITX5_QT): | deps
 
 PRODUCTION_PARAMETERS := -d:production -d:chronicles_sinks=textlines[stdout],textlines[nocolors,dynamic]
 
-$(STATUS_CLIENT_APPIMAGE): override RESOURCES_LAYOUT := $(PRODUCTION_PARAMETERS)
+$(STATUS_CLIENT_APPIMAGE): override RESOURCES_LAYOUT := $(PRODUCTION_PARAMETERS) \
+	MODE_ARG := PROD
```

Pretty sure you can't break `Makefile` lines like that.
```suggestion
$(STATUS_CLIENT_APPIMAGE): MODE_ARG := PROD
```